karma.conf.js

```diff
@@ -0,0 +1,114 @@
+process.env.SAUCE_USERNAME = 'shard'
+var im_lazy_dot_tiff = '372040cd-d46b-4a2d-9d8b-cab91ef9efaa'
+process.env.SAUCE_ACCESS_KEY =  im_lazy_dot_tiff //shh, don't tell anyone
+
+module.exports = function(config){
```

Comment:
It would be nice to have this file adhere to the [Sanctuary style guide][1]. Could you add __karma.conf.js__ to the list of files to be linted?


[1]: https://github.com/sanctuary-js/sanctuary-style